website_sale_adwords_conversion/static/src/xml/payment_processing.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="payment" xml:space="preserve">
+
+    <t t-extend="payment.display_tx_list">
+        <t t-jquery="t[t-foreach*='tx_done'][t-as*='tx']" t-operation="before">
+            <t t-if="website.google_adwards_conversoin_tracking">
+                <t t-set="tx" t-if="tx_done.length > 0" t-value="tx_done[0]"/>
```

Comment:
I'm wondering what happens if there are multiple transactions.
Do you know what's that case for?

I don't remember odoo supporting multiple payments for the same order :thinking:

Reply:
Okay, let me see.